torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2398,22 +2427,28 @@ def produce_guards(
         assert len(placeholders) == len(sources)
         Tensorlike = (torch.Tensor, FakeTensorMeta)
 
+        def _create_no_constraints_policy(t):
+            return FreshCreateSymbolicPolicy(
+                # Ignored; only the constraints part is relevant below.
+                dynamic_sizes=[DimDynamic.DYNAMIC] * t.dim(),
```

Comment:
maybe defensively set this to `None` instead of a list of DimDynamic, if we expect it never to be used?

Reply:
it has to be a list because of `FreshCreateSymbolicPolicy.__post_init__()`, but I could do a list of Nones?

Reply:
ok nevermind haha